### DISCUSSION TITLE
Failed to extract the table without left and right borders.

### DISCUSSION THREAD

**Discussion by sunjingji** [jsvine/pdfplumber#1156] [Ask for help with specific PDFs]
Failed to extract the table without left and right borders， like this：
![table](https://github.com/jsvine/pdfplumber/assets/7960633/49f2bac4-533a-48ef-8f94-9d6847c33178)

**Comment by jsvine** [comment]
Appears to have been handled in https://github.com/jsvine/pdfplumber/issues/1157#event-13195950706
